content/docs/use-cases/versioning-data-and-model-files/tutorial.md

```diff
@@ -1,3 +1,10 @@
+---
+title: 'Tutorial: Data & Model Versioning'
+description: 'Get experience with data versioning in a basic machine learning
+version control scenario: managing multiple datasets and ML model versions using
+DVC commands.'
```

Comment:
Keep in mind we're changing the use case a lot in #1747 — basically rewritten. It has the same ideas but maybe reuse some of the existing text + insert any needed terms?

Reply:
BTW can you please review the text there to make sure we won't lose any traffic from of the terms that current rank high? Thanks

Reply:
Oh never mind, this is the tutorial, I see. We're rewriting the use case only so far.

Reply:
What about the term "dataste versioning" you mentioned in the PR description, should it be included here or featured more in the text? (Currently it's buried under **Automating capturing**).

Reply:
- Why not "hands-on experience" like in the intro?
- Is "machine learning version control" too much? Should the terms be separated (same in the actual text).
- Related: term "version(ing)" is probably repeated too much in this description (3 times).
- End at "using DVC."

Reply:
> can you please review the new text in #1747 to make sure we won't lose any traffic from of the terms that current rank high?

Will do.

> * Why not "hands-on experience" like in the intro?

Trying to keep it under 160 characters, but by removing 'commands' from the end I was able to add this.

> * Is "machine learning version control" too much? Should the terms be separated (same in the actual text).
> * Related: term "version(ing)" is probably repeated too much in this description (3 times).

"machine learning version control" is an important search term that DVC.org already ranks in position 1 for, probably due to mentions like this. So we should keep it if possible. 😉

Agree, 'version(ing)' mentioned three times is too much, so I removed the last instance of 'version' in the description and mirrored in actual text. 'data versioning' and 'machine learning version control' are the two important mentions to keep.

> * End at "using DVC."

Done, and mirrored in actual text.

Reply:
> > BTW can you please review the new text in #1747

> Will do.

Thanks but actually don't worry about this yet. I'll request your review from there when ready.

Reply:
OK about "machine learning version control"!

Thanks for the other updates. Resolving